### DISCUSSION TITLE
Extract DLLs at Runtime

### DISCUSSION THREAD

**Discussion by SeanDugdale** [Excel-DNA/ExcelDna#727] [Q&A] [answered]
Hi there,
I am currently working with ZeroMQ in order to stream data asynchronously into Excel cells. This is working as expected however, I encounter issues when it comes to packaging and installing. I have to pack both the native libzmq.dll as well as the .NET wrapper ZeroMQ.dll however, when I install the 64-bit addin there are exceptions thrown due to not being able to correctly load the native library. I was wondering if there was any way to extract the binding and native dlls to the addin directory or if there is a more appropriate method for this case (we can have the two dlls unpacked but would rather they be packed within the xll for cleanliness). Thanks very much.

**Comment by govert** [comment, marked as answer]
What is the .NET runtime version you are targeting, and the Excel-DNA version you're using?
I think we currently support native library packing only for the .NET core targets, not .NET Framework.

**Reply by SeanDugdale** [reply]
Ahh we're using version 1.7.0 of Excel-DNA from NuGet but are targeting .NET Framework 4.7.2 which I suppose is the issue then, thanks for the info!
